lib/ReactViews/DragDropNotification.tsx

```diff
@@ -0,0 +1,88 @@
+import classNames from "classnames";
+import { reaction } from "mobx";
+import { observer } from "mobx-react";
+import { FC, useEffect, useRef, useState } from "react";
+import { Trans } from "react-i18next";
+import Icon from "../Styled/Icon";
+import { useViewState } from "./Context";
+import Styles from "./drag-drop-notification.scss";
```

Comment:
Suggestion: Would you consider converting to `styled-component` as part of this PR? It doesn't look too complicated to me.

Reply:
Done